pkg/operatorsource/appregistryhelper.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
 	"github.com/operator-framework/operator-marketplace/pkg/appregistry"
+	interface_client "github.com/operator-framework/operator-marketplace/pkg/client"
```

Comment:
sorry one more nit here: Shouldn't this be camelCase instead of snake_case? i.e. `interfaceClient` ?

Reply:
I just copied what was being used previously but happy to change it across

Reply:
addressed

Reply:
The `interface_client` usage was introduced in #147. Changing that across the board should not be done in this PR.

Reply:
ok reverting